sdk/src/validation_status.rs

```diff
@@ -91,6 +91,18 @@ impl ValidationStatus {
         is_success(&self.code)
     }
 
+    // Maps errors into validation_status codes.
```

Comment:
Will we make this list complete?  Not sure it is possible today.

Reply:
Ideally we get rid of the mapping and assign validation status codes when we generate the errors. But I suspect a few cases will always fall through. FYI, there's a new 1.1 approved general.error code we should add in those cases.